com.sublimetext.three.yaml

```diff
@@ -85,7 +87,7 @@ modules:
       - mkdir -p /app/bin
       - mkdir -p /app/lib
       - install apply_extra /app/bin
-      - install sublime.sh /app/bin/sublime
+      - install sublime.sh /app/bin/subl
```

Comment:
I think this could be simplified even further by naming the script `subl` to begin with, then this could be `install subl /app/bin` or folded into the previous line

Reply:
Changed.